src/Sarif.Viewer.VisualStudio/SarifSnapshot.cs

```diff
@@ -125,7 +128,20 @@ public override bool TryGetValue(int index, string columnName, out object conten
                         //url = string.Format("http://www.bing.com/search?q={0} {1}", _errors[index].Provider.Name, _errors[index].ErrorCode);
                     }
 
-                    content = Uri.EscapeUriString(url);
+                    if (url != null)
+                    {
+                        content = Uri.EscapeUriString(url);
+                    }
+                }
+                else if (columnName == StandardTableKeyNames.ErrorCodeToolTip)
+                {
+                    var error = _errors[index];
+                    content = error.RuleId + ":" + error.RuleName;
+                }
+                else if (columnName == StandardTableKeyNames.DetailsExpander)
+                {
+                    var error = _errors[index];
+                    content = !string.IsNullOrEmpty(error.FullMessage);
```

Comment:
OK, I see it now, it's in the new methods you added to implement `IWpfTableEntriesSnapshot`.